### ISSUE TITLE
Multiclass prediction outputs index instead of class label

### ISSUE THREAD

**Issue by Ayaro** [automl/auto-sklearn#46]
For multiclassification task output is an index, which makes result ambiguous, e.g. [0 ... 7] instead of [1 ... 8]

```
class AutoML(BaseEstimator, multiprocessing.Process):
   ...
    def predict(self, X):
        return np.argmax(self.predict_proba(X), axis=1)
```

Does it work as expected?

**Comment by mfeurer** [comment]
You're right, the code is wrong here; it should be something like in [sklearn forests](https://github.com/scikit-learn/scikit-learn/blob/c957249/sklearn/ensemble/forest.py#L478).

**Comment by mfeurer** [comment]
Fixed by #55
